app/controllers/application_controller.rb

```diff
@@ -1,2 +1,64 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
+
+  def role_object( title )
+    Role.find_by(title: title)
+  end
+  helper_method :role_object
+
+  def is_user_a?( role_title )
+    if Permission.where(role_id: role_object( role_title )).exists?
```

Comment:
Keep it DRY. 

_Permission.where(role_id: role_object( role_title ))_ is repeated on line 10 and 11. Please store it in a variable and then use it in both places.

Reply:
Dry is the right way to go